docs/install/knative-with-operators.md

```diff
@@ -82,8 +82,34 @@ kubectl logs -f deploy/knative-operator
 
 1. Create and apply the Knative Serving CR:
 
+      <!-- This indentation is important for things to render properly. -->
+
+   {{< tabs name="serving_cr" default="Install Current Serving (default)" >}}
+   {{% tab name="Install Current Serving (default)" %}} You can install the latest available Knative Serving in the
+   operator by applying the following content of the Serving CR:
+
+```
+apiVersion: v1
+kind: Namespace
+metadata:
+ name: knative-serving
+---
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+  namespace: knative-serving
+```
+
+If you do not specify the version with `spec.version`, the operator will take the latest one available in the operator.
```

Comment:
```suggestion
If you do not specify the version with `spec.version`, the operator default to the latest available version.
```

Reply:
"If you do not specify a version by using `spec.version`, the operator defaults to the latest available version."